selfdrive/athena/athenad.py

```diff
@@ -472,6 +474,32 @@ def log_handler(end_event):
       cloudlog.exception("athena.log_handler.exception")
 
 
+def stat_handler(end_event):
+  while not end_event.is_set():
+    last_scan = 0
+    try:
+      curr_scan = sec_since_boot()
+      if curr_scan - last_scan > 10:
+        for stat_entry in os.listdir(STATS_DIR):
```

Comment:
maybe better to only process one file every ten seconds. That should still allow to work a backlog, but not allow athena to use too much CPU.